packages/SwingSet/src/vats/comms/cdebug.js

```diff
@@ -0,0 +1,11 @@
+let enableCDebug = false;
```

Comment:
global mutable state. hm.

Adding this to the list of things to think about in #2160

Reply:
It's not global.

Reply:
In what way is it not global? `cdebugEnable` is exported, so the flag can be changed from anywhere.

Reading the flag is a little more involved (stupid `toString` tricks) but it's not prevented.